Assets/Scripts/model/main/PeltzerMain.cs

```diff
@@ -723,6 +710,33 @@ void Start()
             // Try to perform the setup. If we fail, that's ok, we'll try again in Update() until we succeed.
             TrySetup();
         }
+        
+        /// <summary>
+        /// Gets the user path depending on the platform.
+        /// </summary>
+        /// <returns>user path used for auto-saving</returns>
+        private string GetUserPath()
+        {
+#if UNITY_ANDROID && !UNITY_EDITOR
+            return Application.persistentDataPath;
```

Comment:
In Open Brush, we store data in /sdcard/Open Brush rather than in the persistentDataPath so that it doesn't get deleted if the user uninstalls the app. We should probably do something similar here to. (we'll need, though, to request some additional permissions)